### PR TITLE
Fix Sanitizer end to end tests

### PR DESCRIPTION
**Pull request by kamilszostak**

### CODE REVIEW

**Comment by dnduffy** [comment]
:shipit:

File: JavaScript/JavaScriptSDK.Tests/TestFramework/PollingAssert.ts [hunk 1]
```diff
@@ -10,9 +10,9 @@ class PollingAssert {
     * @param {number} pollIntervalMs - polling interval in milliseconds
     * @returns {(nextTestStep) => void} callback which will be invoked by the TestClass
     */
-    public static createPollingAssert(assertionFunctionReturnsBoolean: () => boolean, assertDescription: string, timeoutSeconds: number = 30, pollIntervalMs: number = 500) {
```
Comment (MaxShehovtsov) [inline JavaScript/JavaScriptSDK.Tests/TestFramework/PollingAssert.ts]: Is it just indentation changed?

**Comment by MaxShehovtsov** [comment]
:shipit:

File: JavaScript/JavaScriptSDK.Tests/TestFramework/PollingAssert.ts (hunk 1 above)
Comment (kamilszostak) [inline JavaScript/JavaScriptSDK.Tests/TestFramework/PollingAssert.ts]: No, I added a return type - ": (nextTestStep) => void".

**Pull request merged**